releasenotes/notes/threethirteen-d40d659d8939fe5e.yaml

```diff
@@ -1,4 +1,53 @@
 ---
 upgrade:
   - |
-    Makes the library compatible with Python 3.13
+    Makes the library compatible with Python 3.13.
+
+    The following limitations currently apply to support for Python 3.13:
+      - Windows wheels for 3.13 are not built
+      - Appsec Threat Detection is not tested against Django, Flask, or FastAPI with 3.13
+      - Automatic Service Naming is not tested with 3.13
+      - The ``ddtrace-run`` entrypoint is not tested with 3.13
```

Comment:
This might be a 3.13 deal breaker, IMHO.

Reply:
It's fixed here https://github.com/DataDog/dd-trace-py/pull/11793